.github/workflows/test.yml

```
@@ -261,24 +287,35 @@ jobs:
+        shell: bash
         run: |
-          output=$(bun add is-odd --verbose --force 2>&1)
+          status=0
+          output=$(bun add is-odd --verbose --force 2>&1) || status=$?
 
-          if echo "$output" | grep -q "HTTP/1.1 GET https://registry.npmjs.org/is-odd"; then
+          if [ 0 -eq "${status}" ] && grep <<<"$output" -q "HTTP/1.1 GET https://registry.npmjs.org/is-odd"; then
             echo "Successfully installed from default registry"
           else
+            echo "${output}"
+        echo "Return code: ${status}"
```

actionlint (1.7.11)

[error] unexpected key "echo \"Return code" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)